### PR TITLE
p7zip: add page

### PR DESCRIPTION
**Pull request by mt-empty**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

I closed my previous PR #3811 due to different git personas, should be fixed now.
I also added some of andrik's suggestions.

### CODE REVIEW

File: pages/common/p7zip.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# p7zip
+
+> Wrapper of 7-Zip file archiver with high compression ratio.
+> Internally executes either 7za or 7zr command.
```
Comment (mebeim) [inline pages/common/p7zip.md]:
```suggestion
> Internally executes either 7za or 7zr command.
> More information: <http://p7zip.sourceforge.net>.
```

We could add the homepage here.

Comment (andrik) [reply pages/common/p7zip.md]: I totally forgot about that :man_facepalming:

File: pages/common/p7zip.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Archive a file, replacing it with a 7zipped compressed version:
+
+`p7zip {{path/to/file}}`
```
[the remark this answers is not in the document]
Comment (sbrl) [reply pages/common/p7zip.md]: I'm not sure we do this on other pages, @zdroid

File: pages/common/p7zip.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Archive a file keeping the input file:
+
+`p7zip -k {{path/to/file}}`
```
[the remark this answers is not in the document]
Comment (sbrl) [reply pages/common/p7zip.md]: I'm not sure we do this on other pages, @zdroid

File: pages/common/p7zip.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`p7zip -d {{compressed.ext}}.7z`
+
+- Decompress a file keeping the input file:
+
+`p7zip -d -k {{compressed.ext}}.7z`
+
+- Skip some checks and force compression or decompression:
+
+`p7zip -f {{path/to/file}}`
```
[the remark this answers is not in the document]
Comment (sbrl) [reply pages/common/p7zip.md]: I'm not sure we do this on other pages, @zdroid (ref `keepass2`

[the remark this answers is not in the document]
Comment (mt-empty) [reply pages/common/p7zip.md]: I used the instruction in [style-guide.md](https://github.com/tldr-pages/tldr/blob/master/contributing-guides/style-guide.md).
 Ctrl+F {{path/to/file}}

File: pages/common/p7zip.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (mt-empty) [reply pages/common/p7zip.md]: I used the instruction in [style-guide.md](https://github.com/tldr-pages/tldr/blob/master/contributing-guides/style-guide.md).
 Ctrl+F {{path/to/file}}

File: pages/common/p7zip.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (mt-empty) [reply pages/common/p7zip.md]: I used the instruction in [style-guide.md](https://github.com/tldr-pages/tldr/blob/master/contributing-guides/style-guide.md).
 Ctrl+F {{path/to/file}}

[the remark this answers is not in the document]
Comment (zdroid) [reply pages/common/p7zip.md]: You're right, I made a mistake to comment this in a few PRs recently. It's all good now.

**Pull request merged**